.env

```diff
@@ -0,0 +1,2 @@
+REGION = eu_west_2
```

Comment:
isn't it with dashes instead of underscores? please check

Reply:
Yes, it is with dashes. Changed